tests/components/channel_notifications_modal/describe.test.jsx

```diff
@@ -51,7 +51,7 @@ describe('components/channel_notifications_modal/NotificationSection', () => {
     });
 
     test('should match snapshot, on NEVER', () => {
-        const props = {...baseProps, memberNotifyLevel: NotificationLevels.NEVER};
+        const props = {...baseProps, memberNotifyLevel: NotificationLevels.NONE};
```

Comment:
It's NONE not NEVER under `NotificationLevels` constant.

Reply:
Still `NEVER` in the test description.

Reply:
Good catch! I'll update that.